lib/web_ui/lib/src/engine/view_embedder/style_manager.dart

```diff
@@ -116,6 +116,11 @@ void applyGlobalCssRulesToSheet(
     // Hide placeholder text
     '$cssSelectorPrefix .flt-text-editing::placeholder {'
     '  opacity: 0;'
+    '}'
+
+    // Hide outline when the flutter-view root element is focused.
```

Comment:
With the current code, is this even visible? Do we ever leave the focus in the flutter-view root?

Reply:
> With the current code, is this even visible?
Yes it is

> Do we ever leave the focus in the flutter-view root?
Yes, when the widget traversal is done the idea is that we should jump to the next/previous DOM node.